packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1006,14 +1006,35 @@ oie.try.again = Try again
 
 ## Device Enrollment Terminal
 enroll.title.oda = Download Okta Verify
+enroll.title.oda.with.account = Additional setup required to use Okta FastPass
+enroll.title.oda.without.account = Set up an Okta Verify Account
 enroll.title.mdm = Additional setup required
+enroll.subtitle.fastpass = On this device, do you already have an Okta Verify account for {0}?
+enroll.option.noaccount.fastpass = No, I don’t have an account
+enroll.option.account.fastpass = Yes, I already have an account
 enroll.explanation.p1 = To sign in using Okta Verify, you will need to set up Okta Verify on this device.
 enroll.explanation.p2 = In the app, follow the instructions to add an organizational account. When prompted, choose <span class="semi-strong">Sign In</span>, then enter the <span class="semi-strong">sign-in URL</span>:
 enroll.oda.android.step1 = On Google Play, download the Okta Verify app.
 enroll.oda.step1 = Open Okta Verify and follow the steps to add your account.
 enroll.oda.step2 = When prompted, choose Sign In, then enter the sign-in URL:
 enroll.oda.step3 = Download the Okta Verify app.
 enroll.oda.step6 = Finish setting up your account in Okta Verify, then try accessing this app again.
+enroll.oda.with.account.explanation = Okta FastPass is a security method that can sign you in without needing your username.
+enroll.oda.with.account.subtitile1 = Already have Okta FastPass enabled for your account?
+enroll.oda.with.account.subtitile2 = Need to set up Okta FastPass for your account?
+enroll.oda.with.account.step1 = Make sure your Okta Verify is up-to-date, then try signing in again.
+enroll.oda.with.account.step2 = If your device has a work profile, make sure you are signing in from the same profile that Okta Verify is installed in.
+enroll.oda.with.account.step3 = On certain browsers and applications, Okta FastPass is not supported.
+enroll.oda.with.account.step4 = On this device, open the Okta Verify app.
+enroll.oda.with.account.step5 = On the list of accounts, tap your account for {0}.
+enroll.oda.with.account.step6 = Under the “Okta FastPass” section, tap Setup, then follow the instructions.
+enroll.oda.with.account.step7 = After your setup is complete, return here to try signing in again.
```

Comment:
This is a **lot** if information to show on a single view. Can you see how this looks on a small screen resolution?